lib/routing.js

```diff
@@ -7,26 +7,32 @@ if (typeof window !== 'undefined') {
   history = createHistory()
 }
 
-const mapper = new Morph()
+const mapper = new Morph({
+  types: {
+    bool: v => (v ? v === true || v === 'true' : v)
```

Comment:
Thanks for using the type mappings! Can you make it so that the type can only ever map to `true` or `false`? Right now a bunch `null` and `undefined` and `0` would all stay as such and we want to convert those to false.

Maybe something like:
```javascript
v => {
  if (v === 'false') return false
  return Boolean(v)
}
```

Reply:
Also please ignore Travis (:

Reply:
At first I had it that way, but it was interfering with the defaults if you go to the main url without any parameters.